### ISSUE TITLE
Class array-syntax breaks no-unused-selector

### ISSUE THREAD

**Issue by lukashass** [future-architect/eslint-plugin-vue-scoped-css#46]
Using [Array-Syntax](https://vuejs.org/v2/guide/class-and-style.html#Array-Syntax) with an inline `if` breaks reporting of `no-unused-selector`.  
Doing the same with Object-Syntax works fine.

I added two testcases for demonstration (https://github.com/lukashass/eslint-plugin-vue-scoped-css/commit/f0bff82d100b2c944dbdd2ceab4aa280825fdcf4):  

```js
// reports `.bar` as unused
{
    code: `
    <template>
        <div><div :class="{foo: b}"/></div>
    </template>
    <style scoped>
    .foo {}
    .bar {}
    </style>
    <script>
    export default {
        data () {
            return {
                b: true
            }
        }
    }
    </script>
    `,
    errors: [
        {
            messageId: "unused",
            data: { selector: ".bar" },
            line: 7,
            column: 13,
            endLine: 7,
            endColumn: 17,
        },
    ],
},
// should report `.bar` as unused but doesn't
{
    code: `
    <template>
        <div><div :class="[b ? 'foo' : '']"/></div>
    </template>
    <style scoped>
    .foo {}
    .bar {}
    </style>
    <script>
    export default {
        data () {
            return {
                b: true
            }
        }
    }
    </script>
    `,
    errors: [
        {
            messageId: "unused",
            data: { selector: ".bar" },
            line: 7,
            column: 13,
            endLine: 7,
            endColumn: 17,
        },
    ],
},
```

**Comment by ota-meshi** [comment]
Thank you for posting this issue.

The actual cause of this problem is not using array literals, but using ternary expressions.
By using the ternary expression, it is considered untraceable and no error is reported.

To solve this issue, we need to add feature to track simple ternary expressions.

**Comment by wolfgangwalther** [comment]
> By using the ternary expression, it is considered untraceable and no error is reported.

This seems strange to me. Wouldn't it be better to report `.foo` and `.bar` as unused in the 2nd case?

Or at least provide a config option. In a bigger component this would basically disable checking of unused selectors for the whole component if any untraceable syntax is used, right?
